### PR TITLE
Fix junos zuul CI failures

### PR DESCRIPTION
**Pull request by ganeshrn**
##### SUMMARY

*  Fix assertion check to work with zuul CI
   for junos integration testcases

##### ISSUE TYPE

- Testcas Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by Qalthos — APPROVED** [review]

**Review by pabelanger — APPROVED** [review]
This was tested at: 
  https://logs.zuul.ansible.com/74/74/a12f6c87d6ca0ef51e955a4bcb008d11ceda1915/check/ansible-test-network-integration-junos-python36/a85f910/job-output.html

no failures related to this any more

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

File: test/integration/targets/junos_rpc/tests/netconf/rpc.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@
 - assert:
     that:
       - "result.changed == false"
-      - "'<name>\nem0\n</name>' in result['xml']"
+      - "'<interface-information' in result['xml']"
```
Comment (pabelanger) [inline test/integration/targets/junos_rpc/tests/netconf/rpc.yaml:13]: This isn't correct, we need to drop the single quote:

  https://logs.zuul.ansible.com/01/56601/0ddf48a36c67950e70abe358a8bb43fb4a9cf8d9/third-party-check/ansible-test-network-integration-junos-python37/d38da69/controller/ara-report/result/9ce267ed-c1b3-4ff8-9d61-69531b7807d7/

**Review by pabelanger — CHANGES_REQUESTED** [review]

**Comment by pabelanger** [comment]
recheck

**Comment by ganeshrn** [comment]
Fixed by https://github.com/ansible/ansible/pull/57309

**Pull request closed without merging**